redpanda/resources/cluster/data_cluster.go

```diff
@@ -212,70 +431,12 @@ func datasourceClusterSchema() schema.Schema {
 				Computed:    true,
 				Description: "The URL of the cluster API.",
 			},
-			"aws_private_link": schema.SingleNestedAttribute{
-				Computed:    true,
-				Description: "The AWS Private Link configuration.",
-				Attributes: map[string]schema.Attribute{
-					"enabled": schema.BoolAttribute{
-						Computed:    true,
-						Description: "Whether Redpanda AWS Private Link Endpoint Service is enabled.",
-					},
-					"connect_console": schema.BoolAttribute{
-						Computed:    true,
-						Description: "Whether Console is connected in Redpanda AWS Private Link Service.",
-					},
-					"allowed_principals": schema.ListAttribute{
-						ElementType: types.StringType,
-						Computed:    true,
-						Description: "The ARN of the principals that can access the Redpanda AWS PrivateLink Endpoint Service. To grant permissions to all principals, use an asterisk (*).",
-					},
-				},
-			},
-			"azure_private_link": schema.SingleNestedAttribute{
-				Computed:    true,
-				Description: "The Azure Private Link configuration.",
-				Attributes: map[string]schema.Attribute{
-					"allowed_subscriptions": schema.ListAttribute{
-						ElementType: types.StringType,
-						Computed:    true,
-						Description: "The subscriptions that can access the Redpanda Azure PrivateLink Endpoint Service. To grant permissions to all principals, use an asterisk (*).",
-					},
-					"connect_console": schema.BoolAttribute{
-						Computed:    true,
-						Description: "Whether Console is connected in Redpanda Azure Private Link Service.",
-					},
-					"enabled": schema.BoolAttribute{
-						Computed:    true,
-						Description: "Whether Redpanda Azure Private Link Endpoint Service is enabled.",
-					},
-				},
-			},
-			"gcp_private_service_connect": schema.SingleNestedAttribute{
```

Comment:
Yeah, here are all of the data source removals. It sounds like this might have been accidental then?